### PR TITLE
Fixed/auto follow on scroll

### PR DESCRIPTION
**Pull request by Shanky0**
#20 

This PR aims to resolves the auto follow on scroll issue. Please check if i have got all the condition correct.

### CODE REVIEW

File: src/App.svelte [hunk 1]
```diff
@@ -120,15 +131,19 @@
 
 <main>
 	<div class="row">
-		<div class="column" id="message-log">
+		<div class="column" id="message-log" on:wheel={stopFollow}>
```
Comment (glaslos) [inline src/App.svelte:134]: Simplify this handler and in the button using an arrow function expression

**Review by glaslos — CHANGES_REQUESTED** [review]
Nice work so far. A couple of concerns. The placement of the button is fine.

File: src/App.svelte (hunk 1 above)
Comment (Shanky0) [reply src/App.svelte:134]: do i need to just write it as arrow function   as  `()=>stopFollow()` or directly implement it the function here ?

Comment (glaslos) [reply src/App.svelte:134]: You can assign directly in the arrow function

**Comment by Shanky0** [comment]
Sir i have done the changes please check if this does not create conflict for readme. @glaslos

**Comment by shanky-ced** [comment]
> Thank you! LGTM (and we need a Prettier config for formatting).

Yes sir, i on working on it and will soon update readme for windows as well.

**Comment by Shanky0** [comment]
> Thank you! LGTM (and we need a Prettier config for formatting).

Yes sir, i am working on it and also updating readme for windows as well. I have finally able to set it up on windows.

**Pull request merged**